session4/eevee.py

```diff
@@ -0,0 +1,9 @@
+class Eevee:
+    def __init__(self,name,type): 
+        self.name = name
+        self.type = type
+    
+    def growl(self):
+        print(self.name + " used Growl!")
+    def tackle(self):
```

Comment:
Consistency nitpick - add line break between methods
```suggestion
        print(self.name + " used Growl!")

    def tackle(self):
```